### DISCUSSION TITLE
Is Virtualizer stable yet ?

### DISCUSSION THREAD

**Discussion by kevinfairclough** [microsoft/fluentui#30440] [Q&A]
Is Virtualizer stable yet ?

**Comment by ValentinaKozlova** [comment]
According to the documentation it's not https://react.fluentui.dev/?path=/docs/preview-components-virtualizer--default

**Comment by kevinfairclough** [comment]
Yet its being used here https://react.fluentui.dev/?path=/docs/components-combobox--default#virtualizer

**Reply by ValentinaKozlova** [reply]
I don't know why it's being used there. I double-checked, it's still unstable. As I know, it might be moved to https://github.com/microsoft/fluentui-contrib repo. Components in this repo are more flexible from a release strategy

**Reply by kevinfairclough** [reply]
In the meantime we have moved to using third-party library react-window and react-window-infinite-loader with the fluent components.  Do you know if the components are being designed so that they work with other libraries, they seem to now in v9?

**Reply by ValentinaKozlova** [reply]
They should work fine with other libraries
